### PR TITLE
Update 002_plot_components.R

### PR DESCRIPTION
**Pull request by junyangliao**
Added documentation for devs

### CODE REVIEW

File: R/002_plot_components.R [hunk 1]
```diff
@@ -2,7 +2,18 @@
 #
 # Contains functions `create_rawplot_components`, `create_deltaplot_components` and `create_violinplot_components`.
 
-# Function for creation of list of TRUE/FALSE for raw plot components that will be built
+#' Generates list of TRUE/FALSE for raw plot components that will be built
+#' 
+#' This function generates a list of booleans to be passed into the raw plot function.
```
Comment (sunroofgod) [inline R/002_plot_components.R]: Can you reword this to "generates a list of booleans determining whether certain plot components will be constructed for the rawplot."

Its not being passed into the plot_raw function so i figured might be misleading for devs.

File: R/002_plot_components.R [hunk 2]
```diff
@@ -92,7 +120,23 @@ create_deltaplot_components <- function(proportional,
...
+#'
+#' @param boots Boot result obtained from boot.ci
+#' @param idx List of vectors of control-test groupings that determines the arrangement
+#' of the final dataframe output.
+#' @param float_contrast Boolean value determining if a Gardner-Altman plot or 
+#' Cumming estimation plot will be produced.
+#' @param delta_y_max Max y limits for the delta-delta plot
+#' @param delta_y_min Min y limits for the delta-delta plot
+#' @param flow Boolean value determining whether the bars will be plotted in pairs.
+#' @param zero_dot Boolean value determining if the zero dot will be implemented.
```
Comment (sunroofgod) [inline R/002_plot_components.R]: Change "implemented" to "constructed"

File: R/002_plot_components.R [hunk 3]
```diff
@@ -49,7 +60,24 @@ create_rawplot_components <- function(proportional,
   return(plot_component)
 }
 
-# Function for creation of list of TRUE/FALSE for delta plot components that will be built
+#' Generates list of TRUE/FALSE for delta plot components that will be built
+#' 
+#' This function generates a list of booleans to be passed into the delta plot function.
```
Comment (sunroofgod) [inline R/002_plot_components.R]: Same as above but can you reword this to "generates a list of booleans determining whether certain plot components will be constructed for the deltaplot."

**Review by sunroofgod — CHANGES_REQUESTED** [review]
Alls good with the documentation, just particular about certain wordings. Thanks!

File: R/002_plot_components.R (hunk 1 above)
Comment (junyangliao) [reply R/002_plot_components.R]: Done

File: R/002_plot_components.R (hunk 3 above)
Comment (junyangliao) [reply R/002_plot_components.R]: done

File: R/002_plot_components.R (hunk 1 above)
Comment (sunroofgod) [reply R/002_plot_components.R]: Oh oops, i didnt realise there were multiple lines with this: theres line 7 also

Thanks

File: R/002_plot_components.R [hunk 4]
```diff
@@ -60,7 +60,7 @@ create_rawplot_components <- function(proportional,
   return(plot_component)
 }
 
-#' Generates list of TRUE/FALSE for delta plot components that will be built
```
Comment (sunroofgod) [inline R/002_plot_components.R]: I actually think this built way of phrasing is good. Think the one to change is the ones below for the function part.

File: R/002_plot_components.R (hunk 2 above)
Comment (junyangliao) [reply R/002_plot_components.R]: done

File: R/002_plot_components.R (hunk 1 above)
Comment (junyangliao) [reply R/002_plot_components.R]: Done

**Review by sunroofgod — APPROVED** [review]
Looks good! Thanks
